### ISSUE TITLE
Traffic analytics: make it more obvious that you need to enable this feature on the addons tab

### ISSUE THREAD

**Issue by stsewd** [readthedocs/ext-theme#454]
I think we are not enabling analytics by default. If you go to the "Traffic analytics" tab of a project, you'll see an empty list without a clear indication why you are not receiving traffic. We should check if analytics isn't enabled and put a warning or something there.

**Comment by humitos** [comment]
> I think we are not enabling analytics by default

Yes, we decided to make it opt-in because it consumes too many resources in our side and lot of people are not looking at these numbers.
